backend/pkg/api/handlers/public.go

```diff
@@ -610,12 +610,7 @@ func (h *HandlerService) PublicPostValidatorDashboardValidators(w http.ResponseW
 		handleErr(w, r, err)
 		return
 	}
-	var limit uint64
-	if isUserAdmin(userInfo) {
-		limit = math.MaxUint32 // no limit for admins
-	} else if dashboardLimit >= existingValidatorCount {
-		limit = dashboardLimit - existingValidatorCount
-	}
+	limit := dashboardLimit - existingValidatorCount
```

Comment:
Question: Why did you remove the limit for admins?

Reply:
Because I think it's redundant. Realized that this should already be set to `maxJsInt` via https://github.com/gobitfly/beaconchain/blob/BEDS-875/rewards-chart/backend/pkg/commons/db/user.go#L268-L272